frontend/src/common.rs

```diff
@@ -589,40 +589,53 @@ pub fn view_calendar<Ms>(entries: Vec<(NaiveDate, usize, f64)>, interval: &Inter
 
 pub fn view_chart<Ms>(
     labels: &[(&str, usize)],
-    chart: Result<String, Box<dyn std::error::Error>>,
+    chart: Result<Option<String>, Box<dyn std::error::Error>>,
+    no_data_label: Option<String>,
```

Comment:
Do you think we need the flexibility to have custom no-data labels here? A boolean flag would prevent the possibility of accidentally adding inconsistencies.

Reply:
I originally thought that more different labels would be required, but I agree, "No data." and nothing seems to work just fine. I changed the parameter to a bool.